File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/VoiceSourceVO.java
package com.sdkwork.app.model;

public class VoiceSourceVO {
    private String id;
    private String name;
    private String type;
    private String language;
    private Integer sampleRate;

    public String getId() {
        return this.id;
    }
    
    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return this.name;
    }
    
    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return this.type;
    }
    
    public void setType(String type) {
        this.type = type;
    }

    public String getLanguage() {
        return this.language;
    }
    
    public void setLanguage(String language) {
        this.language = language;
    }

    public Integer getSampleRate() {
        return this.sampleRate;
    }
    
    public void setSampleRate(Integer sampleRate) {
        this.sampleRate = sampleRate;
    }
}

File: sdkwork-app-sdk-csharp/Models/VoiceSourceVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VoiceSourceVO
    {
        public string? Id { get; set; }
        public string? Name { get; set; }
        public string? Type { get; set; }
        public string? Language { get; set; }
        public int? SampleRate { get; set; }
    }
}
